src/main/java/com/brew/oauth20/server/repository/ClientsUserRepository.java

```diff
@@ -10,7 +10,11 @@
 import java.util.UUID;
 
 public interface ClientsUserRepository extends JpaRepository<ClientUser, UUID> {
-    @Query(value = "SELECT distinct cu FROM ClientUser cu JOIN FETCH cu.client c WHERE c.clientId = :clientId AND cu.userId = :userId")
+    @Query(value = "SELECT distinct cu FROM ClientUser cu " +
+            "JOIN FETCH cu.client c " +
+            "LEFT JOIN FETCH cu.clientUserScopes cus " +
```

Comment:
Why left join? we don't need client user scopes that not belongs to client user

Reply:
query starts with ClientUser. left join is all about ClientUserScopes. if clientUserScopes is empty we still want ClientUser. That's why.